source/type/data.cpp

```diff
@@ -72,7 +72,7 @@ namespace pt = boost::posix_time;
 namespace navitia {
 namespace type {
 
-const unsigned int Data::data_version = 9;  //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 10;  //< *INCREMENT* every time serialized data are modified
```

Comment:
Do we need this upgrade since there is not any modification on data syrialization ?

Reply:
Exact

Reply:
[Here](https://github.com/CanalTP/navitia/pull/3348/files#diff-347d523f6725669454dd92c1db510fcbd5f8326b7dd2830c779b1629ca3fd18dR101), no ?